cg/meta/transfer/external_data.py

```diff
@@ -191,11 +197,10 @@ def add_transfer_to_housekeeper(
                 bundle_name=sample.internal_id
             )
             fastq_paths_to_add: list[Path] = self.get_fastq_paths_to_add(
-                customer=cust, hk_version=last_version, lims_sample_id=sample.internal_id
+                customer=customer_id, hk_version=last_version, lims_sample_id=sample.internal_id
             )
-            self.add_files_to_bundles(
+            self.add_and_include_files_to_bundles(
```

Comment:
This is a bit problematic since this function commits file to housekeeper and below we only commit if no files fail the md5sum check.

We can probably move it into aforementioned new function, and just not add files that failed the check, instead of not committing any files if one fails